package.json

```diff
@@ -1,9 +1,41 @@
 {
   "name": "react-devstack",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "An ultimate development stack built on top of React with Redux",
   "author": "Tomas Weiss <tomasw@salsitasoft.com>",
   "license": "MIT",
+  "scripts": {
+    "build": "babel src -d lib",
```

Comment:
Babel is used for transpilling sources for this project as well as `babel-loader` for transpiling application.